GAMMA/RTC/README_RTC_GAMMA.txt

```diff
@@ -21,25 +26,28 @@ k:          Not Filtered (n) or Filtered (f)
 l:          Entire Area (e) or Clipped Area (c)
 m:          Dead Reckoning (d) or DEM Matching (m)
 ssss:       Product ID
+```
 
 The source granule used to generate the products contained in this folder is:
 [GRAN_NAME]
 
 -------------
-The side-looking geometry of SAR imagery leads to geometric and radiometric distortions, causing foreshortening, layover, shadowing, and radiometric variations due to terrain slope. Radiometric terrain correction converts unprocessed SAR data into geocoded tiff images with values directly relating to physical properties, alleviating the inherent SAR distortions. The process improves backscatter estimates and provides geolocation information, so images can be used as input for applications such as the monitoring of deforestation, land-cover classification, and delineation of wet snow-covered areas.
+### Using this data ###
 
-Refer to the ASF Sentinel-1 RTC User Guide for additional guidance on the use of this dataset:
-https://asf.alaska.edu/wp-content/uploads/2019/02/Sentinel_RTC_Users_Guide.pdf
+This data was processed by the ASF DAAC HyP3 [YEARPROCESSED] using the HyP3 RTC GAMMA plugin version [HYP3_VER] and GAMMA software [GAMMA_VER].  Contains modified Copernicus Sentinel data [YEARACQUIRED], processed by ESA.
 
-Visit the GAMMA Remote Sensing website for more information about GAMMA SAR software:
-https://www.gamma-rs.ch/
+When using this data in a publication or presentation, we ask you minimally include the following acknowledgement:
 
--------------
-### To cite the data: ###
-ASF DAAC [YEARPROCESSED] using GAMMA software. Contains modified Copernicus Sentinel data [YEARACQUIRED], processed by ESA.
+    RTC products processed by ASF DAAC HyP3 [YEARPROCESSED], contains modified Copernicus Sentinel data [YEARACQUIRED].
```

Comment:
Retain the software credit?

Reply:
I couldn't decide. Maybe:

```
RTC Product processed by ASF DAAC HyP3 [YEARPROCESSED] using GAMMA [GAMMA_VER], contains modified Copernicus Sentinel data [YEARACQUIRED].
```
The wording of the original is hard to drop into a manuscript and I don't want it to get too long. 

I also dropped the `processed by ESA` bit as it's not actually mentioned in the [terms and conditions](https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/sentinel-1-terms-conditions/) but it is included on the [ASF how to cite](https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/sentinel-1-how-to-cite/) page (Note: this page confuses citation and acknowledgement). 

@khogenso do you know if we need the `processed by ESA` bit? Where did that come from?

Reply:
Really, I'd want it to be:

```
RTC product(s) processed by ASF DAAC HyP3 [YEARPROCESSED] using GAMMA [GAMMA_VER], and contains modified Copernicus Sentinel data [YEARACQUIRED] originally processed by ESA
```
Which I think satisfies the terms and conditions but is venturing a bit from the ASF how to cite or the original

Reply:
We should leave the content of the original ESA_citation.txt file exactly as-is, my understanding is that it's something ESA requested.  I mean the sentence: "Contains modified .... processed by ESA"

Reply:
I added the processed by bit back in. Anyone have thoughts on whether we should include the GAMMA version?

Reply:
I don't think we need the version in the acknowledgement sentence. It's in the metadata if people want to dig. The important thing for the acknowledgement is to say that we used their software.

Reply:
Sounds good -- GAMMA version are dubious anyways so just saying software is probably better.